tests/main/abort/task.yaml

```diff
@@ -2,6 +2,7 @@ summary: Check change abort
 
 environment:
     SNAP_NAME: test-snapd-tools
+    SNAPD_NO_MEMORY_LIMIT: 1
```

Comment:
it'd be nice with a comment explaining why it needs the no memory limit

Reply:
agree

Reply:
It is not needed, it was removed, Thanks for catching that.